docs/apis/batch/index.md

```diff
@@ -2123,7 +2123,7 @@ Note that types registered with `registerKryoType()` are not available to Flink'
 
 - `disableAutoTypeRegistration()` Automatic type registration is enabled by default. The automatic type registration is registering all types (including sub-types) used by usercode with Kryo and the POJO serializer.
 
-
+- `setTaskCancellationInterval(long interval)` Sets the the interval (in milliseconds) to wait between consecutive attempts to cancel a running task. By default this is set to **30000** milliseconds, or **30 seconds**.
```

Comment:
I would explain a bit more detailed what's going on. The `cancel()` method is called only once. The interval determines the interrupt frequency!